sktime/forecasting/base/_base.py

```diff
@@ -2540,6 +2540,132 @@ def _get_columns(self, method="predict", **kwargs):
 BaseForecaster._init_dynamic_doc()
 
 
+class BaseGlobalForecaster(BaseForecaster):
```

Comment:
I suppose that the BaseGlobalForecaster should in general also be able to perform probabilistic forecasts. It is okay to add this not directly in this PR I suppose. But this requires tracking of this via an additional issue.

Reply:
@fkiraly any comments on this?

Reply:
agreed - let's try to wrap this up and add proba capability later.